bedrock/config/application.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/** @var string Directory containing all of the site's files */
+$root_dir = dirname(__DIR__);
+
+/** @var string Document Root */
+$webroot_dir = $root_dir . '/web';
+
+/**
+ * Expose global env() function from oscarotero/env
+ */
+Env::init();
+
+/**
+ * Use Dotenv to set required environment variables and load .env file in root
+ */
+$dotenv = new Dotenv\Dotenv($root_dir);
+if (file_exists($root_dir . '/.env')) {
+    $dotenv->load();
+    $dotenv->required(['DB_NAME', 'DB_USER', 'DB_PASSWORD', 'WP_HOME', 'WP_SITEURL']);
+}
+
+/**
+ * Set up our global environment constant and load its config first
+ * Default: production
+ */
+define('WP_ENV', env('WP_ENV') ?: 'production');
+
+$env_config = __DIR__ . '/environments/' . WP_ENV . '.php';
+
+if (file_exists($env_config)) {
+    require_once $env_config;
+}
+
+/**
+ * URLs
+ */
+define('WP_HOME', env('WP_HOME'));
+define('WP_SITEURL', env('WP_SITEURL'));
+
+/**
+ * Custom Content Directory
+ */
+define('CONTENT_DIR', '/app');
+define('WP_CONTENT_DIR', $webroot_dir . CONTENT_DIR);
+define('WP_CONTENT_URL', WP_HOME . CONTENT_DIR);
+
+/**
+ * DB settings
+ */
+define('DB_NAME', env('DB_NAME'));
+define('DB_USER', env('DB_USER'));
+define('DB_PASSWORD', env('DB_PASSWORD'));
+define('DB_HOST', env('DB_HOST') ?: 'localhost');
+define('DB_CHARSET', 'utf8mb4');
+define('DB_COLLATE', '');
+$table_prefix = env('DB_PREFIX') ?: 'wp_';
+
+/**
+ * Authentication Unique Keys and Salts
+ */
+define('AUTH_KEY', env('AUTH_KEY'));
+define('SECURE_AUTH_KEY', env('SECURE_AUTH_KEY'));
+define('LOGGED_IN_KEY', env('LOGGED_IN_KEY'));
+define('NONCE_KEY', env('NONCE_KEY'));
+define('AUTH_SALT', env('AUTH_SALT'));
+define('SECURE_AUTH_SALT', env('SECURE_AUTH_SALT'));
+define('LOGGED_IN_SALT', env('LOGGED_IN_SALT'));
+define('NONCE_SALT', env('NONCE_SALT'));
+
+/**
+ * Custom Settings
+ */
+define('AUTOMATIC_UPDATER_DISABLED', true);
+define('DISABLE_WP_CRON', true);
+define('DISALLOW_FILE_EDIT', true);
+define('WP_DEFAULT_THEME', 'ppo');
```

Comment:
`WP_DEFAULT_THEME` should be `intranet`, or whatever the directory name of your theme is. Although this only applies when the DB is blank, since the active theme is usually specified somewhere in the depths of the `wp_options` table.

Reply:
I assume this would be our child theme name `intranet-theme-clarity` not parent theme?

Reply:
Correct. It should be the directory name of whichever theme you want to be active.